test/b2gedmntuples_cfg.py

```diff
@@ -768,7 +768,7 @@
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
     jetLabel  = cms.InputTag('jetUserDataAK8'),
-    puppiSDjetLabel = cms.InputTag('packedPatJetsAK8PFPuppiSoftDrop'),
+    puppiSDjetLabel = cms.InputTag('slimmedJetsAK8'),
```

Comment:
I also agree with @dmajumder about self consistency, but also is really the POG recipe that strict in the sense that it works ONLY with the subjets from miniAOD? I would assume that the recipe is for the current reconstruction and not for the specific jet collection, but I haven't follow closely this recipe.